src/darwin/Framework/CHIP/MTRBaseDevice.h

```diff
@@ -145,6 +147,7 @@ MTR_EXTERN NSString * const MTREventTimeTypeKey MTR_AVAILABLE(ios(16.5), macos(1
 MTR_EXTERN NSString * const MTREventSystemUpTimeKey MTR_AVAILABLE(ios(16.5), macos(13.4), watchos(9.5), tvos(16.5));
 MTR_EXTERN NSString * const MTREventTimestampDateKey MTR_AVAILABLE(ios(16.5), macos(13.4), watchos(9.5), tvos(16.5));
 MTR_EXTERN NSString * const MTREventIsHistoricalKey MTR_AVAILABLE(ios(17.3), macos(14.3), watchos(10.3), tvos(17.3));
+MTR_EXTERN NSString * const MTRPreviousDataKey MTR_AVAILABLE(ios(17.4), macos(14.4), watchos(10.4), tvos(17.4));
```

Comment:
Shouldn't we duplicate these in MTRDevice? (Or move them to another file?)

Reply:
I originally put this in MTRBaseDevice.h because this is where the data-value dictionary is defined and documented. I will move this to MTRDevice.h and put some documentation around the delegate callback.

Reply:
Done.